R/variable-as-methods.R

```diff
@@ -0,0 +1,143 @@
+#' as.* methods for variables
+#'
+#' Use the as.* family of functions to make a derived copy of a variable that 
+#' has been converted into a new type.
+#' 
+#' Each type of Crunch variable (text, numeric, categorical, etc.) has an `as.*`
+#' function (`as.Text`, `as.Numeric`, and `as.Categorical` respectively) that 
+#' takes the input given as `x`, and makes a new derived variable that is now of 
+#' the type specified. See below for detailed examples.
+#' 
+#' For `as.Text` and `as.Numeric`, aliases to the R-native functions 
+#' `as.character` and `as.numeric` are provided for convenience.
+#'
+#' @param x a Crunch variable to derive and convert to a new type
+#' @param format for `as.Datetime`, when the variable in `x` is a text or 
```

Comment:
I found this a bit confusing, possibly because of the multiple uses of the word "format". Maybe something like "When x is text or categorical, the new date-time variable's format"?